### PR TITLE
Update globalkeyshortcuts.conf

### PR DESCRIPTION
**Pull request by scootergrisen**
Personally i would change "Internet. Web browser." to just "Web browser".
Then the short cut (Ctrl+Alt+I) could be changed to Ctrl+Alt+W.
That would also make it easier to type on a QWERTY layout at least using just one hand.

### CODE REVIEW

File: config/globalkeyshortcuts.conf [hunk 1]
```diff
@@ -31,11 +31,11 @@ Enabled=true
 Exec=pcmanfm-qt
 
 [Control%2BAlt%2BI.7]
-Comment=Internet. Web browser.
+Comment=Web browser
 Enabled=true
-Exec=xdg-open, http://google.com
+Exec=x-www-browser
```
Comment (agaida) [inline config/globalkeyshortcuts.conf]: x-www-browser is debian and derivatives only (inkluding Ubuntu)
so the xdg thing is better imho

**Comment by agaida** [comment]
Maybe we can/should open the systems default browser directly instead of xdg open. Any ideas?

**Comment by scootergrisen** [comment]
Instead of running a specific commando that opens a web browser could we not ask the system indirectly?
So its will be the system that runs the actual command that starts the web browser.
But i don't know how i just think its a bit silly if its not possible.
Maybe check other software to see how they do it.

**Comment by tsujan** [comment]
Here `bash -c $BROWSER` (as the Exec of a desktop file) opens the default broswer but I can't guarantee hat it'll work everywhere.

I confirm that there's no `x-www-browser` on Arch.

**Comment by yan12125** [comment]
Here's [how Qt's openUrl() detects the browser](https://github.com/qt/qtbase/blob/5b1f18b0f1cd07f2dfcafe79b456338f908de531/src/platformsupport/services/genericunix/qgenericunixservices.cpp#L84-L117). It tries xdg-open first.

**Comment by scootergrisen** [comment]
Maybe someone could investigate when happens when the user presses the web key that some keyboards have that opens a browser.
Maybe we could emulate a press on that key?

**Comment by tsujan** [comment]
> Here's how Qt's openUrl() detects the browser. It tries xdg-open first.

Yes, because xdg-open opens urls with the default browser. Then, the code checks for EVs $BROWSER and $DEFAULT_BROWSER.

@agaida, @scootergrisen, @yan12125 
Doesn't `bash -c $BROWSER` open the default browser for you?

**Comment by scootergrisen** [comment]
$BROWSER and $DEFAULT_BROWSER is not set on my system.
```
LANG=c bash -c $BROWSER
bash: -c: option requires an argument
LANG=c bash -c $DEFAULT_BROWSER
bash: -c: option requires an argument

```

**Comment by agaida** [comment]
@scootergrisen - try echo $BROWSER - but if not set, it is not set :)
```
% echo $BROWSER
firefox
```

**Comment by tsujan** [comment]
> $BROWSER and $DEFAULT_BROWSER is not set on my system.

If so, the best way of doing it is what was done before, namely `Exec=xdg-open http://google.com`.  @yan12125's link confirms that too.

**Comment by tsujan** [comment]
In short, I think `Exec=xdg-open http://google.com` shouldn't change.

**Comment by scootergrisen** [comment]
Why force the users of Xfce to use a specific company like google?

**Comment by tsujan** [comment]
> Why force the users of Xfce to use a specific company like google?

You mean LXQt, of course. There's no force. Users could change shortcuts. For example, I've removed the browser shortcut.

Why google? The same question could be asked about any other engine. Moreover, and for example, I don't think any programmer uses another engine when searching for codes; there's simply no time for shortcomings of other search engines. But please see that this is just *my* opinion and I have no intention to defend it. IMO, such a simple matter dosn't deserve much discussion.

**Comment by agaida** [comment]
good point - maybe some  neutral link like https://www.microsoft.com or https://oracle.com would fit better - just kidding.

I'm fine with google as it is a fallback when no other valid url is available - maybe i'm doing something wrong but mostly i type browser and url in terminal, klick on a link in a terminal and so on - not to forget that  the shortcut can be changed without problems in the ui

**Comment by scootergrisen** [comment]
Why not link to xfce.org if lets say in 2016 it is not possible to start a web browser without opening a website at the same time.
That would at least have something to do with Xfce.

Saying all programmers use google is like saying all computer users use Windows.

I'm sick of being forced feed google.
Let the user decide instead of forcing the user.

**Comment by agaida** [comment]
@scootergrisen  you are not forced, you can change the link - please stop this

**Comment by scootergrisen** [comment]
Sure i will spend my time translating other projects instead.

**Comment by agaida** [comment]
@scootergrisen - we really don't want link to xfce - that would be nuts as we are LXQt - and both links would provide no better user experience - i'm not the greatest google-fan ever - but it should be a good search engine. https://startpage.com or https://duckduckgo.com comes to mind - if we could consder one of them as good choice we should change the link - and if so - we should use https

**Comment by yan12125** [comment]
Removing Google links is a great idea. However, using ```x-www-browser``` is wrong, as confirmed by @agaida and @tsujan. I'm not blaming you - it does not make sense to ask a contributor test his/her proposals on all possible platforms before submitting a pull request. @scootergrisen As you're the author of this pull request, **you** (not _us_, LXQt developers) are responsible for coming up with a good solution for your idea.

> Maybe we could emulate a press on that key?

> Maybe check other software to see how they do it

Both are good start points. Just go ahead and implement it.

By the way, please study the difference of Xfce and LXQt. Proposing Xfce stuffs on a LXQt place again and again is not respectful.

**Comment by scootergrisen** [comment]
So i wrote Xfce and xfce.org instead of LXQt and lxqt.org by mistanke.
Anyway i will just stop making suggestions.
Problem solven.

**Comment by tsujan** [comment]
@yan12125 @agaida 
Please respect @scootergrisen's latest choice!

**Comment by yan12125** [comment]
Hmmm leaving LXQt? I'm a little bit sad but I can't do anything more.

**Comment by palinek** [comment]
Ad: opening the "empty" browser window/tab

I think we can happily use the `Exec=xdg-open about:blank`, as this is detected as URL by [xdg-open](https://cgit.freedesktop.org/xdg/xdg-utils/tree/scripts/xdg-open.in#n71):
```sh
# Returns true if argument is a file:// URL or path
is_file_url_or_path()
{
    if echo "$1" | grep -q '^file://' \
            || ! echo "$1" | egrep -q '^[[:alpha:]+\.\-]+:'; then
        return 0
    else
        return 1
    fi
}
```
and as so opened by the browser.

What do you (all) think?

**Comment by agaida** [comment]
about:blank would be fine for me - we talked short about and it doesn't work - maybe a typo. Imho the best solution.

**Comment by tsujan** [comment]
`about:blank` works here too and should work everywhere. But, frankly, I think this is about an obsession, not a real problem.

**Comment by palinek** [comment]
> But, frankly, I think this is about an obsession, not a real problem.

Hm... I didn't realize, that it is already changed to `duckduckgo.com` in master :)

**Comment by agaida** [comment]
@palinek - that was the second best choice - take a privacy saving search engine - but about:blank would be better i guess. @tsujan google is worth a flamewar and over-reactions everytime, Last time i see in in debians devel list - it was the fallback to 8.8.8.8 in systemd-networkd. And it was really heatened. The outcoming was #nameserver 8.8.8.8 - better nothing than google, so it seems to be a common obsession.

**Comment by yan12125** [comment]
```xdg-open about:blank``` has bugs(?)
https://github.com/lxde/lxqt-common/pull/69#issuecomment-267291430

**Comment by tsujan** [comment]
@yan12125 
Apart from what's written in that comment, have you encountered any problem with xdg-open yourself? Or do you know of any valid bug report?

**Comment by yan12125** [comment]
> have you encountered any problem with xdg-open yourself?

No. I don't use xdg-open.

> Or do you know of any valid bug report?

No

I guess scootergrisen has messed up his/her environment variables. Here's my try:
```
$ XDG_CURRENT_DESKTOP=GNOME xdg-open about:blank
gvfs-open: about:blank: error opening location: The specified location is not supported
```
So in LXQt it should be safe to continue with ```xdg-open about:blank``` and fix any new bugs.

**Comment by tsujan** [comment]
Good! And I agree with you about that error message.

By the way:

>  I don't use xdg-open.

I don't think an indirect usage of xdg-open could be avoided easily. I mean that you use it but not directly ;)

**Pull request closed without merging**